client/messages/de.json

```
@@ -1144,12 +1177,17 @@
   "ciDaNU": "Automatisch den ersten Seitenaufruf beim Laden des Skripts erfassen",
   "KzO2sB": "Tracking des ersten Seitenaufrufs aktiviert",
   "FcSY6x": "Tracking des ersten Seitenaufrufs deaktiviert",
+  "5zGub0": "Fehlerverfolgung",
```

Biome (2.4.12)

[error] The key 5zGub0 was already declared.

(lint/suspicious/noDuplicateObjectKeys)